make/conf/jib-profiles.js

```diff
@@ -680,20 +679,46 @@ var getJibProfilesProfiles = function (input, common, data) {
             common.debug_profile_artifacts(artifactData[name]));
     });
 
-    profilesArtifacts = {
-        "linux-x64": {
+    buildJdkDep = input.build_os + "-" + input.build_cpu + ".jdk";
+    docsProfiles = {
+        "docs": {
+            target_os: input.build_os,
+            target_cpu: input.build_cpu,
+            dependencies: [
+                "boot_jdk", "devkit", "graphviz", "pandoc", buildJdkDep,
+            ],
+            configure_args: [
+                "--enable-full-docs",
+                "--with-build-jdk=" + input.get(buildJdkDep, "home_path")
+                    + (input.build_os == "macosx" ? "/Contents/Home" : "")
```

Comment:
I'm not fully understanding the platform selection mechanism here. Previously, docs were only built on linux-x64. I can see no such filtering in the new code; and this piece here even indicates assumed support for macosx.

Question: Is docs still built only on one platform? How is this achieved then?

Reply:
I wanted to make the jib profile "docs" buildable on any platform (though I guess windows is likely not going to work well). This makes future development more flexible for local builds. The platform we choose to use for our official  CI/tier configurations is controlled elsewhere and will continue to be linux-x64.